client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
@shroman  

Whether It has a  problem or not which depends on how user understand its purpose. In my point of view, my understanding would be:  user specified is always prior to dymatically fetched, which means if I specify, JUST DO NOT change it anymore since I have totolly understand the which nameserver I am using, I don't want to risk of changing my nameservers without my notice.

Sure, somebody's understanding would be: user specified is prior to dymatically fetch, however, this only happens when my specified nameserver address is available. 

I would say it is also reasonable. Maybe we can provide a policy for them to choose.